assemblyline_v4_service/updater/helper.py

```diff
@@ -135,8 +135,11 @@ def url_download(source: Dict[str, Any], previous_update: int, logger: Logger, o
                 else:
                     headers = {'If-Modified-Since': previous_update}
 
-            if fetch_method in ['get', 'post']:
-                response = getattr(session, fetch_method)(uri, auth=auth, headers=headers, proxies=proxies, stream=True)
+            if fetch_method == 'get':
+                response = session.get(uri, auth=auth, headers=headers, proxies=proxies, stream=True)
+            elif fetch_method == 'post':
+                json = source.get('post_data') or None
```

Comment:
In the future, we may need to detect if `source.get('post_data')` is proper json, because session.post() could take a data parameter instead of a json one.